sys/include/net/gnrc/ipv6/blacklist.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
```

Comment:
Just wondering: was it intentional that you mention @authmillenon here instead of your name?